src/Form/PetForm.php

```diff
@@ -47,6 +48,14 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'class' => Species::class,
                 'choice_label' => 'speciesName',
             ])
+            ->add('gender', ChoiceType::class, [
```

Comment:
what does this have to do with search?

Reply:
I wanted to have more criteria for search and in the first commit I added gender and breed.

Reply:
Still doesn't justify why is this part of this PR and not a new one